journals/apps/journals/blocks.py

```diff
@@ -16,9 +18,17 @@
 STREAM_DATA_TYPE_FIELD = 'type'
 STREAM_DATA_DOC_FIELD = 'doc'
 
+SPAN_ID_FORMATTER = '{doc_type}-{doc_id}'
+BLOCK_FORMATTER = '{span} {block}'
+
+
+def get_unique_span(doc_type, doc_id):
```

Comment:
can we move it in `utils.py` where we have defined `make_md5_hash` and also change its signature to something like this
`get_block_prefix(block_type, block_id)`

Reply:
yes, Nice Idea

Reply:
`get_block_prefix(block_type, block_id)` is more related to blocks. should not it to remain in **blocks.py** ?